src/components/Loans/ClosedLoans/ClosedLoans.js

```diff
@@ -36,7 +36,7 @@ import Label from '../../Label';
 import ErrorModal from '../../ErrorModal';
 
 import css from './ClosedLoans.css';
-import { localizeCurrencyAmount } from '../../util/localizeCurrencyAmount';
+import localizeCurrencyAmount from '../../util/localizeCurrencyAmount';
```

Comment:
after updating `src/hooks/index.js` file on the lines of other hooks exports (as suggested below in one of the comments), adjust this import